src/one_dragon_qt/app/devtools/image_stitching_interface.py

```
@@ -286,23 +293,17 @@ def on_load_second_clicked(self):
                 self._update_button_states()
 
             except Exception as e:
-                QMessageBox.warning(self, '错误', f'加载第二张图失败: {str(e)}')
+                QMessageBox.warning(self, gt('错误'), f"{gt('加载第二张图失败: ')}{e}")
 
-    def on_direction_changed(self, text: str):
+    def on_direction_changed(self, _index: int):
         """处理拼接方向改变事件。
 
         当用户选择不同的拼接方向时，更新内部状态并记录日志。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)